src/main/resources/config/liquibase/data/fda_submission.csv

```diff
@@ -1,10 +1,10 @@
 id|number|supplement_number|device_name|generic_name|date_received|decision_date|description|curated|genetic|companion_diagnostic_device_id|type_id
-1|P140020|S020|BRACAnalysis CDx|cancer-related germline gene mutation detection system|2019-12-06 00:00:00|2020-05-19 00:00:00|Approval to expand the intended use of BRACAnalysis® CDx to include a companion diagnostic indication for BRCA1/2 mutations in patients with metastatic castration resistant prostate cancer who may benefit from treatment with Lynparza® (olaparib).|FALSE|TRUE|1|1
```

Comment:
I'm curious on how this file is generated and why all the trademarks have been removed.

Reply:
This will be restored in this ticket: https://github.com/oncokb/oncokb/issues/3520